src/routes/email/email.service.ts

```diff
@@ -76,8 +78,16 @@ export class EmailService {
   async getEmail(args: {
     chainId: string;
     safeAddress: string;
-    signer: string;
+    signer: `0x${string}`;
+    authPayload: AuthPayload | undefined;
   }): Promise<Email> {
+    if (
+      !args.authPayload ||
+      args.authPayload.signer_address !== args.signer ||
```

Comment:
This is the comparison responsible for the aforementioned checksumming requirements.